frontend/storage/models/File.ts

```diff
@@ -0,0 +1,21 @@
+export type FileProps = Required<File>;
+export enum Types {
```

Comment:
Rename `File` to `Image`, remove the `type` column and this enum, I'll do the same changes later in the backend